modules/ROOT/pages/policies-landing-page.adoc

```diff
@@ -4,7 +4,46 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: policy, custom, ootb, offline
 
-A policy extends the functionality of an API and enforces certain capabilities such as security. A policy can control access and traffic without modification to the API implementation. Custom policies can provide other capabilities, such as encrypting and logging.
+Policies enable you to enforce regulations to help manage security, control traffic, and improve adaptability of your APIs. For example, a policy can control authentication, access, allotted consumption, and service level access (SLA). You can implement all these regulations with no modification to the code implementation. Mulesoft provides ready-to-use default policies that are shipped with the product. Additionally, you can also create custom policies based on your specific business requirements.
+
+You can apply policies to these types of APIs:
+
+* An APIkit project
++
+For example, deploy the APIkit project to Anypoint Platform using API Autodiscovery and apply a policy.
++
+* An API running on CloudHub
++
+Design an API on Anypoint Platform, configure a proxy for Cloudhub, and apply a policy.
+* An API deployed to a private or cloud-based Mule Runtime 3.8.x or later
++
+* An API deployed to API Gateway Runtime 2.x
+
+ You deploy a proxy API application and apply one or more policies to control how and when a received request is forwarded to its implementation endpoint. You can set an API alert to notify you when an API request violates a policy for SLA. By default, a policy applies to the entire API, filtering traffic requests to every resource and method. In Mule Runtime 3.8.0 and later, you can enhance security through policies by using Gatekeeper. Gatekeeper disables an API until all online policies are applied.
+
```

Comment:
This section is very outdated: 

* API Gateway Runtime 2.x: has been deprecated, we should not mention it. 
* I would change this to say that you can apply a policy to any API implemented with Mulesoft, be RAML/Rest based, HTTP or SOAP based as long as it is exposed through an HTTP endpoint.

You can also apply a policy to APIs not implemented with Mulesoft by deploying a Proxy application to control how and when a received request is forwarded to its implementation endpoint.

Reply:
Done! Thanks!